packages/types/scripts/genereate-web-models.ts

```diff
@@ -0,0 +1,121 @@
+import { EnumDeclaration, Project, SourceFile } from "ts-morph";
```

Comment:
I'm not sure what this file is for.

Reply:
Also I think it should be `generate-web-models.ts` small typo in the name

Reply:
It's for generating TypeScript models (interfaces) out of the open-api spec.
As this generator has some irritating "nits" about how it generates enums, I used `ts-morph` to change single-value-enum to be consist of a single member called `value`